docs/documentation/backwards-compatibility.md

```diff
@@ -42,11 +41,11 @@ In Atom, press **cmd shift F**. It looks like this:
 
 1. If your previous Prototype Kit version was older than 6.3.0, follow the guidance below about updating from older versions.
 
-## Updating pages to use version 7
+## Updating pages to use versions 7 and above
 
-You can now create new pages using version 7 by working in the app/views folder.
+You can now create new pages using version 7 and above by working in the app/views folder.
 
-You can update old pages by moving them from app/v6/views to app/views. You will then need to update the code on these pages to work with version 7. See the [updating your code guide in the GOV.UK Design System](https://design-system.service.gov.uk/get-started/updating-your-code/).
+You can update old pages by moving them from app/v6/views to app/views. You will then need to update the code on these pages to work with version 7 and above. See the [updating your code guide in the GOV.UK Design System](https://design-system.service.gov.uk/get-started/updating-your-code/).
```

Comment:
`You can update old pages by moving them from app/v6/views to app/views. You will then need to update the code on these pages to work with versions 7 and above. See the [updating your code guide in the GOV.UK Design System](https://design-system.service.gov.uk/get-started/updating-your-code/).`